templates/dart/.github/workflows/publish.yml.twig

```diff
@@ -0,0 +1,14 @@
+name: Publish to pub.dev
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
Do we need to specify regex on tags here? Generally "on release published" is enough

Reply:
Yes flutter and dart have this for pub publishing this is official dart lang action
here for reference: https://dart.dev/tools/pub/automated-publishing

Reply:
Interesting, since this regex is invalid and will also match many unexpected cases 😅 it should probably be like:

```suggestion
      - '[0-9]+\.[0-9]+\.[0-9]+.*'
```